website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -39,17 +39,19 @@ Consul supports **four major Envoy releases** at the beginning of each major Con
 
 | Consul Version      | Compatible Envoy Versions                                                          |
 | ------------------- | -----------------------------------------------------------------------------------|
-| 1.18.x              | 1.28.0, 1.27.2, 1.26.6, 1.25.11                                                    |
-| 1.17.x              | 1.27.2, 1.26.6, 1.25.11, 1.24.12                                                   |
-| 1.16.x              | 1.26.6, 1.25.11, 1.24.12, 1.23.12                                                  |
+| 1.18.x              | 1.28.1, 1.27.3, 1.26.7, 1.25.11                                                    |
+| 1.17.x              | 1.27.3, 1.26.7, 1.25.11, 1.24.12                                                   |
+| 1.16.x              | 1.26.7, 1.25.11, 1.24.12, 1.23.12                                                  |
+| 1.15.x              | 1.28.1, 1.27.3, 1.26.7, 1.25.11, 1.26.7, 1.25.11, 1.24.12, 1.23.12                 |
```

Comment:
```suggestion
| 1.15.x (enterprise only)              | 1.28.1, 1.27.3, 1.26.7, 1.25.11, 1.26.7, 1.25.11, 1.24.12, 1.23.12                 |
```

Since 1.15.x LTS is enterprise only we should make this more apparent. This change is specific to when 1.18.0 GA, which is when LTS takes effect so should not be backported past 1.18.x.

Reply:
@david-yu makes sense. WDYT about adding "LTS" to this to clarify further? maybe
```suggestion
| 1.15.x (LTS - Enterprise only) | 1.28.1, 1.27.3, 1.26.7, 1.25.11, 1.26.7, 1.25.11, 1.24.12, 1.23.12                 |
```

Reply:
Looks good. Thank you. 🚢